sql/telemetry/nondesktop_clients_last_seen_v1/view.sql

```diff
@@ -1,37 +1,70 @@
 CREATE OR REPLACE VIEW
-  `moz-fx-data-shared-prod.telemetry.nondesktop_clients_last_seen_v1` AS
+  `moz-fx-data-shared-prod.telemetry.nondesktop_clients_last_seen_v1`
+AS
 -- For context on naming and channels of Fenix apps, see:
 -- https://docs.google.com/document/d/1Ym4eZyS0WngEP6WdwJjmCoxtoQbJSvORxlQwZpuSV2I/edit#heading=h.69hvvg35j8un
 WITH glean_union AS (
-    SELECT
-    * REPLACE('beta' AS normalized_channel),
+  -- Fenix apps
+  SELECT
+    * REPLACE ('beta' AS normalized_channel),
     'Firefox Preview' AS app_name,
   FROM
     `moz-fx-data-shared-prod.org_mozilla_fenix.baseline_clients_last_seen`
   UNION ALL
   SELECT
-    * REPLACE('nightly' AS normalized_channel),
+    * REPLACE ('nightly' AS normalized_channel),
     'Firefox Preview' AS app_name,
   FROM
     `moz-fx-data-shared-prod.org_mozilla_fenix_nightly.baseline_clients_last_seen`
   UNION ALL
   SELECT
-    * REPLACE('release' AS normalized_channel),
+    * REPLACE ('release' AS normalized_channel),
     'Fenix' AS app_name,
   FROM
     `moz-fx-data-shared-prod.org_mozilla_firefox.baseline_clients_last_seen`
   UNION ALL
   SELECT
-    * REPLACE('beta' AS normalized_channel),
+    * REPLACE ('beta' AS normalized_channel),
     'Fenix' AS app_name,
   FROM
     `moz-fx-data-shared-prod.org_mozilla_firefox_beta.baseline_clients_last_seen`
   UNION ALL
   SELECT
-    * REPLACE('nightly' AS normalized_channel),
+    * REPLACE ('nightly' AS normalized_channel),
     'Fenix' AS app_name,
   FROM
     `moz-fx-data-shared-prod.org_mozilla_fennec_aurora.baseline_clients_last_seen`
+  -- Other apps that send Glean telemetry
+  UNION ALL
+  SELECT
+    *,
+    'Lockwise Baseline' AS app_name,
```

Comment:
Why suffix these with `Baseline` instead of following the convention with the Fenix apps?

Reply:
Fenix is the only Glean-based app that is included in KPIs. I needed a way of making the rest of these identifiable with a common prefix or suffix so we can filter them out in the smoot query for the Any Mobile Activity aggregation.

Reply:
We need a more coherent story around how we present app names as captured in https://github.com/mozilla/bigquery-etl/issues/861